platform/android/polyOut/bin/main/eu/polypoly/pod/android/polyOut/FetchInit.kt

```diff
@@ -0,0 +1,7 @@
+package eu.polypoly.pod.android.polyOut
```

Comment:
These changes are a bit suspicious - why does it add these files? It's some unused code that we may or may not have removed in the mean time, but I don't see how it's related to this issue to add them (back).

Reply:
my bad on this, it wasn't meant to be pushed